{{cookiecutter.repo_name}}/.github/workflows/ci.yml

```diff
@@ -13,6 +13,7 @@ jobs:
       - name: Set up Python environment
         uses: actions/setup-python@v4
         with:
+          fetch-depth: 0
```

Comment:
Isn't fetch-depth only available on `actions/checkout` and not on `actions/setup-python`?

Reply:
Ja,  dat klopt.   fetch-depth is nodig voor git-versioning.   nieuw issue aangemaakt: #21